ios/reactNativeWithMoxy.xcodeproj/xcshareddata/xcschemes/reactNativeWithMoxy.xcscheme

```diff
@@ -10,7 +10,7 @@
             ActionType = "Xcode.IDEStandardExecutionActionsCore.ExecutionActionType.ShellScriptAction">
             <ActionContent
                title = "Run Script"
-               scriptText = "echo &quot;.env.dev&quot; &gt; /tmp/envfile&#10;">
+               scriptText = "echo &quot;.env&quot; &gt; /tmp/envfile&#10;">
```

Comment:
Following our discussion [above](https://github.com/moxystudio/react-native-with-moxy/pull/18#discussion_r403095139), please remove this scheme because it is no longer relevant.